ui/raidboss/data/02-arr/raid/t9.js

```diff
@@ -435,7 +452,7 @@
         'Fireball': 'Feuerball',
         'Ghost': 'Geist',
         'Golem Meteors': 'Golem Meteore',
-        'Heavensfall': 'Himmelssturz',
+        'Heavensfall': 'Himmelsfall',
```

Comment:
Was this intentional?

Reply:
omg i'll open request again. thanks

Reply:
I was about to do a release, so I'll fix these up.  No worries!

(Sorry for not noticing this myself.)

Reply:
Umm already requested, how can I cancel it?..